CONTRIBUTING.md

```diff
@@ -9,42 +9,71 @@ Every good PR usually consists of:
 - feature implementation :)
 - documentation to describe this feature to other people
 - tests to ensure everything is implemented correctly
- 
 
 ## Step-by-step guide
 
-### Before the PR (NEED LINKS)
+### Before the PR
 Please ensure that you have read the following docs:
-- [documentation and FAQ]()
-- [minimal examples section]()
+- [changelog](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)
+- [documentation](https://etna-docs.netlify.app/)
+- [tutorials](https://github.com/tinkoff-ai/etna/tree/master/examples)
+
+### Setting up your development environment
+
+Before writing any code it is useful to set up a development environment.
+1. Clone etna library to some folder and go inside:
+```bash
+git clone https://github.com/tinkoff-ai/etna.git etna/
+cd etna
+```
+2. Run installation with `poetry` ([poetry installation guide](https://python-poetry.org/docs/#installation)):
+```bash
+poetry install -E all-dev
+```
```

Comment:
We should probably add section about creating new virtual environment.

Reply:
Something like: we recommend creating new virtual environment. At etna team we use pyenv, but you can use whatever tools you like.

Reply:
As I remember, poetry creates environment by itself by default. May be we should add that to activate this environment contributor should run `poetry shell`?

Reply:
Right! 
We use pyenv to set proper python version. 
So no section about creating new virtual environment is needed.

Reply:
This conversation may be resolved.